iam.tf

```diff
@@ -31,7 +31,16 @@ resource "aws_iam_role" "sns_feedback_role" {
   path                  = var.sns_topic_feedback_role_path
   force_detach_policies = var.sns_topic_feedback_role_force_detach_policies
   permissions_boundary  = var.sns_topic_feedback_role_permissions_boundary
-  assume_role_policy    = data.aws_iam_policy_document.sns_feedback[0].json
+  assume_role_policy = jsonencode({
+    Version = "2012-10-17",
```

Comment:
part 2:
Replace the permissions [here](https://github.com/terraform-aws-modules/terraform-aws-notify-slack/blob/30cd80cc19e3bc459c875fac657715baa49cbeed/iam.tf#L8-L23) with the permissions defined below

also, I guess this has been broken for 2 years?! 😅

Reply:
I am hoping you meant extend this with the inline permission that I had added ?